lib/Doctrine/ORM/Proxy/Factory/StaticProxyFactory.php

```diff
@@ -112,6 +121,8 @@ public function getProxy(string $className, array $identifier) : GhostObjectInte
                 ->entityManager
                 ->getUnitOfWork()
                 ->getEntityPersister($metadata->getClassName());
+        $toManyFields = $this->cachedToManyFields[$className]
```

Comment:
Blank line yo

Reply:
Same group of assignments, hence no added newline